### DISCUSSION TITLE
[Cloud Messaging] Reset badge number functionality

### DISCUSSION THREAD

**Discussion by lvlrSajjad** [invertase/react-native-firebase#6334] [General]
I couldn't find this functionality in the docs. 
It'd be cool if you added a function which resets the badge number.
When user opens the app.

**Comment by mikehardy** [comment]
Hi there! To set expectations clearly: This will not happen. The functionality you request is "on-device notification" functionality, and firebase includes no APIs for that, so this module will include no APIs for that.

You probably want https://notifee.app/

But you should realize that the expectation this will work for Android will lead to disappointment. Android does not include any APIs for badge counts so even Notifee includes no APIs for badge counts on Android. They do exist for iOS though https://notifee.app/react-native/docs/ios/badges
